latest_plugin_versions.json

```
@@ -9,8 +9,8 @@
   "router": {
     "repository": "https://github.com/apollographql/router",
     "versions": {
-      "latest-1": "v1.61.0",
-      "latest-2": "v2.0.0"
+      "latest-1": "v2.1.0",
```

Renovate is properly updating latest-2 to router 2.1.0, however, unfortunately it put it in the same PR that tries to update latest-1 to 2.1.0, which we DO NOT want to do.

```suggestion
      "latest-1": "v1.61.0",
```